_data/simple-icons.json

```diff
@@ -918,6 +918,11 @@
             "hex": "FA243C",
             "source": "https://www.apple.com/itunes/marketing-on-music/identity-guidelines.html#apple-music-icon"
         },
+        {
+            "title": "Apple News",
+            "hex": "FD3159",
```

Comment:
The color should be `#F03355`. The color is from the subscribe button on https://www.apple.com/apple-news/.

Please keep your issue content updated to date.

Reply:
The icon resource is required. Please provide.

Reply:
> The color should be `#F03355`. The color is from the subscribe button on https://www.apple.com/apple-news/.
> 
> Please keep your issue content updated to date.

It is linear gradient from `#FD3159` to `#FD5464`. For this situation, how should the regular work?

Reply:
> It is linear gradient from `#FD3159` to `#FD5464`. For this situation, how should the regular work?

Where can I see the gradient values? Could you provide the resource that you picked from?

Reply:
> > It is linear gradient from `#FD3159` to `#FD5464`. For this situation, how should the regular work?
> 
> Where can I see the gradient values? Could you provide the resource that you picked from?

It seems I made a mistake, sorry...